src/Mautic/Config.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Escopecz\MauticFormSubmit\Mautic;
+
+/**
+ * Mautic Configuration
+ */
+class Config
+{
+    /**
+     * Curl verbose logging option
+     *
+     * @var int
+     */
+    protected $curlVerbose; 
```

Comment:
```suggestion
    protected $curlVerbose = 1; 
```
Let's set the default option to 1 to avoid a BC break.